beacon-chain/blockchain/execution_engine.go

```diff
@@ -439,6 +439,9 @@ func (s *Service) removeInvalidBlockAndState(ctx context.Context, blkRoots [][32
 			// Blobs may not exist for some blocks, leading to deletion failures. Log such errors at debug level.
 			log.WithError(err).Debug("Could not remove blob from blob storage")
 		}
+		if err := s.dataColumnStorage.Remove(root); err != nil {
+			log.WithError(err).Debug("Could not remove data columns from data column storage")
+		}
```

Comment:
Is debug level appropriate for this. In the case of blobs, it is a probable scenario that doesn't need the attention of the user. I just want to confirm that is the case here too. If so, can you add a comment similar to that on line 439?

Reply:
Fixed in 2e01412760ce6249808593f98a2015db2b350aaf.

For the data column case, there is no "expected" error. So writing a log in `error` instead of `debug`.